Wiki/weight-tracker.md

```diff
@@ -1,30 +1,25 @@
-# Weight-tracker
-Weight Tracker is a [Script API](script-api.md) showcase present in the [demo document](document.md).
+# Weight-tracker
 
-[Day notes](day-notes.md) shows (among others) how we have "weight" [promoted attribute](promoted-attributes.md) in the day note [template](template.md). This then aggregates the data and shows a nice chart of weight change in time.
 
-Demo
-----
+The `Weight Tracker` is a [Script API](script-api.md) showcase present in the [demo document](document.md).
 
-![](images/weight-tracker.png)
+By adding `weight` as a [promoted attribute](promoted-attributes.md) to the [Day notes](day-notes.md) through its [template](template.md), you can aggregate the data and plot weight change over time.
```

Comment:
~the~ **d**ay notes (either capitalize both as a Day Notes being a concept, or neither - day notes as instances of the concept)

Or this might be more clear: 
By adding `weight` as a [promoted attribute](promoted-attributes.md) in the  [template](template.md) from which [day notes](day-notes.md) are created,  you can [...]